client/mainmenu/CHighScoreScreen.cpp

```diff
@@ -280,6 +291,8 @@ void CHighScoreInputScreen::clickPressed(const Point & cursorPosition)
 
 void CHighScoreInputScreen::keyPressed(EShortcut key)
 {
+	if(key == EShortcut::HIGH_SCORES_STATISTIC)
+		return;
```

Comment:
Why is this check needed? Add comment perhaps?

Reply:
Any key on keyboard should skip video, except the shortcut.

Btw.: It's also an exotic shortcut (`.`). Because of a drawback of our shortcut system... Can not use key filtering (because every key should skip). But if you use an already assigned Shortcut it gives you back the incorrect shortcut.